### DISCUSSION TITLE
help-install .deb on SECON Desktop?

### DISCUSSION THREAD

**Discussion by Caldersson** [Security-Onion-Solutions/securityonion#12470] [General] [answered]
My organization does not allow Putty, and I am attempting to install SecureCRT.deb to meet their requirements. SECON servers are air-gapped along with the SECON Desktop. When I attempt to install SecureCRT.deb it states dpkg is missing. What can I do to get .deb software to install on an air-gapped SECON desktop?

**Comment by reyesj2** [comment, marked as answer]
You need to use the rpm package of the software you want to install.
